source/developers/experience-builder.rst

```diff
@@ -354,7 +355,7 @@ The complete FreeMarker template for the carousel component becomes:
 
 .. TODO Speak about the ice support classes, event capture overlay and special treatment for empty zones
 
-.. _macros:
+.. _xbMacros:
```

Comment:
@alhambrav I've renamed this ref (macros was way too vague). I did try to find references to it and found none but I don't know rst that well to know if there are syntaxes I might have not checked. Please help me make sure all references are updated appropriately.